### PR TITLE
Update hybrid-agent.md

### PR DESCRIPTION
**Pull request by reikor0x**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md [hunk 1]
```diff
@@ -99,6 +99,9 @@ Keep the following issues in mind before you install the Hybrid Agent:
 
 - The Hybrid Agent supports a single migration endpoint with the service default limits. Multiple migration endpoints utilizing your custom endpoint/URL is not supported.
 
+- The Hybrid Agent supports a single Exchange Organization. Multiple Exchange Orgazation Hybrid is not supported.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md:102]:
```suggestion
- The Hybrid Agent supports a single Exchange organization. Multiple Exchange organization hybrid is not supported.
```

**Comment by reikor0x** [comment]
Pull request updated. Thanks

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by reikor0x** [comment]
branch updated

**Pull request merged**